### ISSUE TITLE
Indented text within input (discussion?)

### ISSUE THREAD

**Issue by dwieeb** [chalk/wrap-ansi#16]
I have a question on some behavior I've noticed with text within long input strings containing newlines.

```javascript
const chalk = require('chalk');
const wrapAnsi = require('wrap-ansi');

const input = `
The quick brown ${chalk.red('fox jumped over')} the lazy ${chalk.green('dog and then ran away with the unicorn.')}

Afterwards, the fox quoted Shakespeare:

    "If music be the food of love, play on."

`.trim();

console.log(wrapAnsi(input, 20));
```

![image](https://user-images.githubusercontent.com/236501/26896301-deca82ac-4b89-11e7-9d4b-63476c170ddf.png)

I guess I would expect the whitespace (multiple spaces, tabs) to be maintained. That, however, may be outside the scope of this module. (What to do when the indented sentence is wrapped?--Are the subsequent lines of the indented sentence indented or not indented?) 

One might say the burden is on the caller. Input should be split on newlines and word wrapped individually and re-concatenated. One might also call this a bug of `wrap-ansi`. 

This may open a discussion for the availability of a higher-level module that has options for parsing and wrapping complex bodies of text, instead of simply expecting single sentences.

**Comment by SamVerschueren** [comment]
Is this what you'd expect it to be? 

<img width="155" alt="screen shot 2017-07-23 at 12 48 07" src="https://user-images.githubusercontent.com/1913805/28498746-321325ba-6fa5-11e7-94ac-3aecfcfb76bc.png">

Because then it's a duplicate of #9 and it will be fixed by PR #17.

**Comment by dwieeb** [comment]
It is. Thanks!
